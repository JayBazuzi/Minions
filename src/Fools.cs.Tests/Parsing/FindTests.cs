using ApprovalTests.Reporters;
using NUnit.Framework;
using Fools.cs.Tests.Support;

namespace Fools.cs.Tests.Parsing
{
	[TestFixture, UseReporter(typeof(DiffReporter))]
	public class FindTests
	{
		[Test]
		public void should_locate_test_cases()
		{
			@"
specify feature the test framework:
	requires a simple empty test should pass:
		pass
".find_blocks().should_parse_correctly();
		}

	}
}
