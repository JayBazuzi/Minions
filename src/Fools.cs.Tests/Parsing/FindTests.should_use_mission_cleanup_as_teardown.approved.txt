Should pass

/** Declarations **/
[
  {
    "feature": "the test framework",
    "body": [
      {
        "name": "clean up",
        "special_purpose": 2,
        "body": []
      },
      {
        "requirement": "a simple empty test should pass",
        "body": []
      }
    ]
  }
]

/** Errors **/
[]
